package.json

```diff
@@ -28,21 +28,21 @@
     "test": "test/run.sh"
   },
   "dependencies": {
-    "@babel/core": "^7.0.0-beta.51",
-    "@babel/runtime": "^7.0.0-beta.51",
-    "@babel/types": "^7.0.0-beta.51",
+    "@babel/core": "^7.0.0",
+    "@babel/runtime": "^7.0.0",
+    "@babel/types": "^7.0.0",
     "commoner": "^0.10.8",
     "private": "^0.1.6",
     "recast": "^0.15.3",
-    "regenerator-preset": "^0.12.0",
-    "regenerator-runtime": "^0.12.1",
-    "regenerator-transform": "^0.13.3",
+    "regenerator-preset": "file:packages/regenerator-preset",
+    "regenerator-runtime": "file:packages/regenerator-runtime",
+    "regenerator-transform": "file:packages/regenerator-transform",
```

Comment:
This changes were made by lerna I think.

Reply:
Should we revert the lerna changes?